skore/src/skore/sklearn/cross_validation/plots/timing_plot.py

```diff
@@ -0,0 +1,125 @@
+"""Plot cross-validation timing results."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    import plotly.graph_objects
+
+
+def plot_cross_validation_timing(cv_results: dict) -> plotly.graph_objects.Figure:
+    """Plot the timing results of a cross-validation run.
+
+    Parameters
+    ----------
+    cv_results : dict
+        The output of scikit-learn's cross_validate function.
+
+    Returns
+    -------
+    plotly.graph_objects.Figure
+        A plot of the time-related cross-validation results.
+    """
+    from datetime import timedelta
+
+    import pandas
+    import plotly
+    import plotly.graph_objects as go
+
+    _cv_results = cv_results.copy()
+
+    # Remove irrelevant keys
+    to_remove = [key for key in _cv_results if key not in ["fit_time", "score_time"]]
+    for key in to_remove:
+        _cv_results.pop(key, None)
+
+    df = pandas.DataFrame(_cv_results)
+
+    dict_labels = {
+        "fit_time": "fit_time (seconds)",
+        "score_time": "score_time (seconds)",
+    }
+
+    def linspace(lo, hi, num):
+        interval = (hi - lo) / (num - 1)
+        return [lo + k * interval for k in range(0, num)]
```

Comment:
Same as above could you use [`numpy.linspace`](https://numpy.org/doc/2.1/reference/generated/numpy.linspace.html) ?